3rd_party_slots/python_client/hyperon_das/commons/properties.py

```diff
@@ -1,3 +1,4 @@
+UINT32_MAX = 2**32 - 1
 
 
 class Properties:
```

Comment:
This class must be a `dict`, eg:
```
class Properties(dict[str, str | int | float | bool]):
```
Or even better (`from collections import UserDict`):
```
class Properties(UserDict[str, str | int | float | bool]):
```